extra/2-piping.js

```diff
@@ -16,26 +16,29 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
-}
-
-function multiply() {
-
-}
-
-function format() {
-
-}
-
+function add(a, b) {
+  return  (a + b)
+  }
+  
+  function multiply(a, b) {
+    return (a * b);
+  }
+  
+  function format(num) {
+    return `£${num}`;
+  }
+  
+  
 const startingValue = 2;
 
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode =
+let badCode = `£${(startingValue + 10) * 2}`;
 
+  
 /* BETTER PRACTICE */
 
-let goodCode =
+let goodCode = format(multiply(startingValue + 10, 2));
+
```

Comment:
:) you forgot about `add()` :) 
```js
format(multiply(add(startingValue,10), 2));
```
but this is also might be difficult to read.
in cases like this one, maybe better to have:
```js
let sum = add(startingValue,10);
let finalResult = format(multiply(sum, 2));
```